types/maxmsp/index.d.ts

```diff
@@ -1219,6 +1223,69 @@ declare class Maxobj {
+     */
+    getattr(attribute_name: "assistshowspatchername" | "bottomtoolbarpinned" | "boxanimatetime" | "enablehscroll"
+        | "enablevscroll" | "fontface" | "fontsize" | "gridonopen" | "gridsnaponopen" | "helpsidebarclosed" | "isolateaudio"
+        | "lefttoolbarpinned" | "objectsnaponopen" | "openinpresentation" | "righttoolbarpinned" | "showontab" | "tallnewobj"
+        | "textjustification" | "toolbarvisible" | "toptoolbarpinned" | "workspacedisabled"): number;
+    getattr(attribute_name: "defaultfocusbox" | "description" | "digest" | "filepath" | "fontname"
+        | "globalpatchername" | "style" | "subpatcher_template" | "tags" | "title"): string;
+    getattr(attribute_name: "accentcolor" | "bgcolor" | "clearcolor" | "color"
+        | "editing_bgcolor" | "elementcolor" | "locked_bgcolor" | "patchlinecolor"
+        | "selectioncolor" | "stripecolor" | "textcolor" | "textcolor_inverse"): Color;
+    getattr(attribute_name: "openrect"): Rect;
+    getattr(attribute_name: "bgfillcolor"): any;
+    getattr(attribute_name: "gridsize"): Size;
+
+    /**
+     * Sets the value of the attribute specified by attribute_name.
+     */
+    setattr(attribute_name: "assistshowspatchername" | "bottomtoolbarpinned" | "boxanimatetime"
+        | "enablehscroll" | "enablevscroll" | "fontface" | "fontsize" | "gridonopen" | "gridsnaponopen"
+        | "helpsidebarclosed" | "isolateaudio" | "lefttoolbarpinned" | "objectsnaponopen" | "openinpresentation"
+        | "righttoolbarpinned" | "showontab" | "tallnewobj" | "textjustification" | "toolbarvisible"
+        | "toptoolbarpinned" | "workspacedisabled", anything: number): void;
+    setattr(attribute_name: "defaultfocusbox" | "description" | "digest" | "filepath" | "fontname"
+        | "globalpatchername" | "style" | "subpatcher_template" | "tags" | "title", anything: string): void;
+    setattr(attribute_name: "accentcolor" | "bgcolor" | "clearcolor" | "color" | "editing_bgcolor"
+        | "elementcolor" | "locked_bgcolor" | "patchlinecolor" | "selectioncolor" | "stripecolor"
+        | "textcolor" | "textcolor_inverse", anything: Color): void;
+    setattr(attribute_name: "openrect", anything: Rect): void;
+    setattr(attribute_name: "bgfillcolor", anything: any): void;
+    setattr(attribute_name: "gridsize", anything: Size): void;
+
+    /**
+     * Returns an Array value containing the names of available attributes for the object's box.
+     */
+    getboxattrnames(): string[];
+
+    /**
+     * Returns the value of the object's box attribute specified by box_attribute_name. Lists are returned as JS Array objects.
+     */
+    getboxattr(box_attribute_name: "background" | "fontface" | "fontsize" | "hidden" | "ignoreclick" | "presentation" | "textjustification" | "number"): number;
+    getboxattr(box_attribute_name: "fontname" | "hint" | "jspainterfile" | "style" | "varname"): string;
```

Comment:
missing `js: getboxattr(box_attribute_name: "annotation"): string;`

Reply:
That's right thanks